ocs_ci/templates/workloads/cephfs_stress/cephfs_filesystem_monitor.py

```
@@ -0,0 +1,633 @@
+        current_time = time.time()
+        
+        # Check if it's time to run
+        if current_time - last_run_time < MONITOR_FULL_FS_INTERVAL:
+            time.sleep(10)
+            continue
+            
+        last_run_time = current_time
+        logger.info(f"[{monitor_name}] Starting full filesystem scan")
+        
+        # Command 1: OPTIMIZED - Use find instead of ls -lRf (10x faster)
+        logger.info(f"[{monitor_name}] Running: find /mnt -type f (optimized)")
```

Ruff (0.15.12)

[warning] Logging statement uses f-string

(G004)